sem/parallelrunner.py

```diff
@@ -1,20 +1,27 @@
 from .runner import SimulationRunner
-from multiprocessing import Pool
+from .utils import CallbackBase
+from multiprocessing.pool import ThreadPool as Pool
```

Comment:
this was taken from StackOverflow right? What about explaining with a few lines the reasoning behind this?

Reply:
ThreadPool should share the memory with the main process, which is important to correctly use the callbacks.
The main difference between with Pool is that Pool spawn 5 independent processes (https://stackoverflow.com/questions/46045956/whats-the-difference-between-threadpool-vs-pool-in-the-multiprocessing-module).